libs/service-logging/src/service_logging/middleware.clj

```diff
@@ -50,9 +50,10 @@
      (tc/with-logging-context {:response (cheshire/encode (clean-response (assoc response
                                                                            :uri uri
                                                                            :request-method request-method)))}
-                              (log/log 'AccessLogger level throwable (str method " " uri)))))
+                              (log/log 'AccessLogger level throwable (str method " " uri))
+                              (tc/clear-ext-svc-tag!))))
```

Comment:
We can't know that a given request is an agave request.
If I were to wrap with that macro where the interaction with agave takes place, the key would be removed from the context before the response is logged.

Therefore, you have to add to the context, and explicitly remove from the context afterward. We also can't rely of the previous uses of that macro to remove the key.